var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -144,15 +148,9 @@ def patch(self):
 
     def build_args(self, spec, prefix):
         args = []
-
         # From NumPy 1.10.0 on it's possible to do a parallel build.
         if self.version >= Version('1.10.0'):
-            # But Parallel build in Python 3.5+ is broken.  See:
-            # https://github.com/spack/spack/issues/7927
-            # https://github.com/scipy/scipy/issues/7112
-            if spec['python'].version < Version('3.5'):
-                args = ['-j', str(make_jobs)]
```

Comment:
I wonder if we should still keep this. What version of Numpy fixed the problem?

Reply:
It was tagged for the `1.13.0` release as per `https://github.com/numpy/numpy/pull/9050#issuecomment-300209782`.

So, should I switch the statement to `if self.version >= Version('1.13.0'): ` instead ?